src/main/scala/org/broadinstitute/hail/driver/Main.scala

```diff
@@ -16,6 +16,32 @@ import scala.collection.JavaConverters._
 import scala.collection.mutable
 import scala.reflect.ClassTag
 
+object SparkManager {
+  var _sc: SparkContext = _
+  var _sqlContext: SQLContext = _
+
+  def createSparkContext(appName: String, master: String): SparkContext = {
+    if (_sc == null) {
+      val conf = new SparkConf().setAppName(appName)
+      conf.setMaster(master)
+      conf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
```

Comment:
It's not the default.  According to this:

https://spark.apache.org/docs/latest/tuning.html

"By default, Spark serializes objects using Java’s ObjectOutputStream framework"